README.md

```diff
@@ -301,6 +301,14 @@ Populate your environment with real school data. This is taken from
 rake data:schools:import
 ```
 
+To get trust data:
```

Comment:
There should be a rake task for this as well, the same as we do for importing schools

Reply:
Yes, there should be

Reply:
Indeed, there is

Reply:
I can't see it. Reverting commit.